src/features/defi/components/GetStarted/GetStarted.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, VStack } from '@chakra-ui/react'
+import osmosis from 'assets/osmosis.svg'
+import { Text } from 'components/Text'
+
+import { DefiModalHeader } from '../DefiModalHeader/DefiModalHeader'
+
+type GetStartedProps = {
+  assetId: string
+}
+
+const CAIP19_TO_MAX_APR = {
+  'cosmoshub-3/slip44:118': '12'
+}
+
+export const GetStarted = ({ assetId: caip19 }: GetStartedProps) => {
+  // TODO: parentheses are nice but let's get asset name from selectAssetByCAIP19 instead of this
+  const asset = (_ => ({
```

Comment:
do you just want to go ahead and write a `selectAssetNameById` selector here? we're going to need it elsewhere.

Reply:
Done ✅